tests/istio.mk

```diff
@@ -69,7 +69,7 @@ e2e_simple: istioctl generate_yaml e2e_simple_run
 
 e2e_simple_cni: istioctl
 e2e_simple_cni: export ENABLE_ISTIO_CNI=true
-e2e_simple_cni: export EXTRA_HELM_SETTINGS=--set istio-cni.excludeNamespaces={} --set istio-cni.pullPolicy=IfNotPresent --set istio-cni.tag=$(ISTIO_CNI_DOCKER_TAG) --set istio-cni.hub=$(ISTIO_CNI_DOCKER_HUB)
+#TODO need to create the CNI helm charts and install the CNI
```

Comment:
shoudln't this just be `e2e_simple_cni: run_daemon` ?

Reply:
Yes - more or less that is what is needed,  but I didn't want to spend to much time trying to get it working in this change as I want to get the gate 
test running for a baseline